index.html

```diff
@@ -14,9 +14,9 @@ <h1>Nabla API sample app</h1>
 
         <div class="filters">
             Demo type: 
-            <a href="#" id="ambient-encounter" class="active">Ambient encounter</a>
+            <a href="#" id="ambient-encounter-link" class="active">Ambient encounter</a>
              | 
-            <a href="#" id="dictated-note" class="">Dictated note</a>
+            <a href="#" id="dictated-note-link" class="">Dictated note</a>
```

Comment:
the id was already used by another element?

Reply:
Yes: https://github.com/nabla/sample-app/blob/main/index.html#L119